scripts/create_bridges

```diff
@@ -21,16 +21,24 @@ from haas.drivers.driver_tools.vlan import get_vlan_list
 
 config.load()
 
+def ip_link(args):
+    return call(['ip', 'link'] + args)
+
 def create_bridge(vlan_no):
     vlan_no = str(vlan_no)
     trunk_nic = cfg.get('headnode', 'trunk_nic')
     bridge = 'br-vlan%s' % vlan_no
     vlan_nic = '%s.%s' % (trunk_nic, vlan_no)
-    call(['brctl', 'addbr', bridge])
-    call(['vconfig', 'add', trunk_nic, vlan_no])
-    call(['brctl', 'addif', bridge, vlan_nic])
-    call(['ifconfig', bridge, 'up', 'promisc'])
-    call(['ifconfig', vlan_nic, 'up', 'promisc'])
+    ip_link(['add', 'type', 'bridge', 'name', bridge])
+    ip_link(['set', 'promisc', 'on', 'dev', bridge])
```

Comment:
> Required for traffic to go through to machines that aren't the host

Are you sure we need it; that proxy ARP doesn't provide the necessary
functionality?

The current commands seem to work so I'd be surprised if we need to
enable promiscuous mode.